### ISSUE TITLE
show-invisibles crashes with Qt6

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#2164]
```
Traceback (most recent call last):
  File "c:\leo.repo\leo-editor\leo\core\leoKeys.py", line 2457, in callAltXFunction
    func(event)
  File "c:\leo.repo\leo-editor\leo\core\leoGlobals.py", line 247, in commander_command_wrapper
    method(event=event)
  File "c:\leo.repo\leo-editor\leo\commands\commanderEditCommands.py", line 1079, in showInvisibles
    showInvisiblesHelper(c, True)
  File "c:\leo.repo\leo-editor\leo\commands\commanderEditCommands.py", line 1102, in showInvisiblesHelper
    frame.body.set_invisibles(c)
  File "c:\leo.repo\leo-editor\leo\plugins\qt_frame.py", line 1506, in set_invisibles
    option.setFlags(QtGui.QTextOption.ShowTabsAndSpaces)
AttributeError: type object 'QTextOption' has no attribute 'ShowTabsAndSpaces'
```

**Comment by tbpassin** [comment]
Confirmed for PyQt6.11.

**Comment by edreamleo** [comment]
Fixed at 37df6f9.  Notes:

1. Tested with both PyQt5 and PyQt6.
2. The definitions in leoQt4/5/6.py are not used.
